app/controllers/rubcop_test_controller.rb

```diff
@@ -1,17 +1,18 @@
-class RubocopController  < ApplicationController
-	def initialize
-		print "this is so bad patern"
-	end
+class RubocopController < ApplicationController
+  def initialize
+    print 'this is so bad patern'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.